### PR TITLE
HBASE-28057 Run spotless:apply and fix any existing spotless issues

### PR DESCRIPTION
**Pull request by NihalJain**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 19s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m  0s |  spotbugs executables are not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 48s |  master passed  |
| +1 :green_heart: |  compile  |   0m  9s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m  8s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m  7s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 11s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 10s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 10s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m  4s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  javadoc  |   0m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 36s |  hbase-hbck2 in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m  6s |  The patch does not generate ASF License warnings.  |
|  |   |   7m 55s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-137/1/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/137 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs checkstyle compile |
| uname | Linux 8da7bcc01db7 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / 0bf6ee1 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-137/1/testReport/ |
| Max. process+thread count | 1257 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 U: hbase-hbck2 |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-137/1/console |
| versions | git=2.30.2 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 1]
```diff
@@ -613,8 +613,7 @@ private static String getCommandUsage() {
 
   private static void usageAddFsRegionsMissingInMeta(PrintWriter writer) {
     writer.println(" " + ADD_MISSING_REGIONS_IN_META_FOR_TABLES + " [OPTIONS]");
-    writer.println("      [<NAMESPACE|"
-      + "NAMESPACE:TABLENAME>...|-i <INPUTFILES>...]");
+    writer.println("      [<NAMESPACE|" + "NAMESPACE:TABLENAME>...|-i <INPUTFILES>...]");
```
Comment (Apache9) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:616]: Here we could just remove the '+' operator?

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 2]
```diff
@@ -767,8 +766,7 @@ private static void usageReplication(PrintWriter writer) {
   }
 
   private static void usageExtraRegionsInMeta(PrintWriter writer) {
-    writer.println(" " + EXTRA_REGIONS_IN_META + " [<NAMESPACE|"
-      + "NAMESPACE:TABLENAME>...|");
+    writer.println(" " + EXTRA_REGIONS_IN_META + " [<NAMESPACE|" + "NAMESPACE:TABLENAME>...|");
```
Comment (Apache9) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:769]: Ditto.

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 3]
```diff
@@ -800,8 +798,8 @@ private static void usageExtraRegionsInMeta(PrintWriter writer) {
   }
 
   private static void usageReportMissingRegionsInMeta(PrintWriter writer) {
-    writer.println(" " + REPORT_MISSING_REGIONS_IN_META + " [<NAMESPACE|"
-      + "NAMESPACE:TABLENAME>...|");
+    writer
+      .println(" " + REPORT_MISSING_REGIONS_IN_META + " [<NAMESPACE|" + "NAMESPACE:TABLENAME>...|");
```
Comment (Apache9) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:802]: Ditto. At least we could remove the last '+'.

Comment (NihalJain) [reply hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:802]: Right. Let me fix these.

**Comment by NihalJain** [comment]
Updated PR!

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 27s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m  0s |  spotbugs executables are not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 49s |  master passed  |
| +1 :green_heart: |  compile  |   0m 10s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m  7s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m  7s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 10s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m  9s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m  9s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m  5s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  javadoc  |   0m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 38s |  hbase-hbck2 in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m  6s |  The patch does not generate ASF License warnings.  |
|  |   |   7m  6s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-137/2/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/137 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs checkstyle compile |
| uname | Linux 8b12821a9e0d 5.4.0-1103-aws #111~18.04.1-Ubuntu SMP Tue May 23 20:04:10 UTC 2023 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / 0bf6ee1 |
| Default Java | Oracle Corporation-1.8.0_342-b07 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-137/2/testReport/ |
| Max. process+thread count | 1282 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 U: hbase-hbck2 |
| Console output | https://ci-hbase.apache.org/job/HBase-Operator-Tools-PreCommit/job/PR-137/2/console |
| versions | git=2.30.2 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by petersomogyi — APPROVED** [review]

**Pull request merged**